add keybindings for draw mode changes and trash

chriswhong opened the pull request:
Adds keybindings:

- When editing `delete` will trigger `draw.trash()`, removing a selected feature or vertex
- When not editing, 
- - `m` will change the draw mode to `draw_point`
- - `l` will change the draw mode to `draw_line_string`
- - `p` will change the draw mode to `draw_polygon`
- - `r` will change the draw mode to `draw_rectangle`
- - `c` will change the draw mode to `draw_rectangle`

Also fixes site.css where some merge conflict markers were accidentally committed.

chriswhong commented:
Note, if #748 makes it in, we will also need to add a keybinding for the circle mode.

wboykinm commented:
> Note, if #748 makes it in, we will also need to add a keybinding for the circle mode.

Wanna merge #748 and add that first here?

chriswhong commented:
@wboykinm This is good to go now, added keybinding for circle mode, also added the shortcuts in the `title` for each draw tool (e.g. 'Draw Rectangular Polygon (r)')

wboykinm commented:
> added the shortcuts in the title for each draw tool (e.g. 'Draw Rectangular Polygon (r)')

Great thinking.

wboykinm commented on css/site.css:
```diff
@@ -102,11 +102,7 @@ p.intro-hint:hover {
 }
 
 .top .buttons {
-<<<<<<< HEAD
```
@chriswhong Looks like another merge artifact here.

wboykinm reviewed (approved):
@chriswhong Awesome!
